staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -327,22 +322,41 @@ var (
 			},
 		},
 	)
-	SuggestedFlowSchemaWorkloadLeaderElection = newFlowSchema(
-		"workload-leader-election", "leader-election", 200,
+	// We add an explicit rule for endpoints-controller with high precedence
+	// to ensure that those calls won't get catch by the following
```

Comment:
s/catch/caught/

Reply:
@MikeSpreitzer I fixed the typo in #107054

Reply:
done